tests/results/test_disease_observer.py

```diff
@@ -261,3 +272,59 @@ def test_different_results_per_disease(base_config, base_plugins):
             "transition_count_human_cortico_deficiency",
         ]
     )
+
+
+@pytest.mark.parametrize(
+    "person_time_exclusions, transition_count_exclusions",
+    [
+        ([], []),
+        (["susceptible_to_human"], ["susceptible_to_human_to_turning"]),
+        (["susceptible_to_human", "turning"], []),
+    ],
+)
+def test_category_exclusions(
+    vampiris, base_config, base_plugins, person_time_exclusions, transition_count_exclusions
+):
+    """Test that we can exclude diseases via the model spec."""
+    vampiris_observer = DiseaseObserver(vampiris.cause)
+
+    # Add exclusions to model spec
+    base_config.update(
+        {
+            "stratification": {
+                "excluded_categories": {
+                    "person_time_vampiris": person_time_exclusions,
+                    "transition_count_vampiris": transition_count_exclusions,
+                }
+            }
+        }
+    )
+    simulation = InteractiveContext(
+        components=[
+            TestPopulation(),
+            vampiris,
+            ResultsStratifier(),
+            vampiris_observer,
+        ],
+        configuration=base_config,
+        plugin_configuration=base_plugins,
+    )
+
+    simulation.step()
+    person_time = simulation.get_results()["person_time_vampiris"]
+    transition_count = simulation.get_results()["transition_count_vampiris"]
+    person_time_categories = {
+        "susceptible_to_human",
+        "turning",
+        "vampire",
+    }
+    transition_count_categories = {
+        "susceptible_to_human_to_turning",
+        "turning_to_vampire",
+    }
+    assert set(person_time["sub_entity"]) == person_time_categories - set(
+        transition_count_exclusions
```

Comment:
Why is this using the `transition_count_exclusions`? Shouldn't it use the `person_time_exclusions`? Same question below.

Reply:
Oof, scary - good catch